### PR TITLE
Add ac_poitiers, fix failing tests

### LINKED ISSUE #144: New ent
Hello,
can you add this ent ?
ac_nantes

thanks

### LINKED ISSUE #145: ENT Poitiers
Hello,
Could you add my ENT, if that's possible ?
The link:
https://ent.ac-poitiers.fr/
Thank you very much

### LINKED ISSUE #147: Failing tests `TestInformation`
The response for `PageActualites` has changed. The demo website does not have any published messages, so I'm not able to see the new structure.

### PR DESCRIPTION
**Pull request by Bapt5**
* add ac_poitiers (fix #145)
* add new structure for Information (fix #147)
* remove non-existant ENT that makes the tests fail
* close #144 

I add the new structure for Information like on the demo but on my personal Pronote it's still the old structure of Information. I think that schools will do the update next year.

### CODE REVIEW

File: pronotepy/dataClasses.py [hunk 1]
```diff
@@ -932,13 +932,17 @@ def __init__(self, client: _ClientBase, json_dict: dict) -> None:
             Util.datetime_parse, "dateCreation", "V"
         )
         self.start_date: datetime.datetime = self._resolver(
```
Comment (bain3) [inline pronotepy/dataClasses.py:934]:
```suggestion
        self.start_date: Optional[datetime.datetime] = self._resolver(
```

File: pronotepy/dataClasses.py [hunk 2]
```diff
@@ -932,13 +932,17 @@ def __init__(self, client: _ClientBase, json_dict: dict) -> None:
...
-            Util.datetime_parse, "dateDebut", "V"
+            Util.datetime_parse, "dateDebut", "V", strict=False
         )
         self.end_date: datetime.datetime = self._resolver(
```
Comment (bain3) [inline pronotepy/dataClasses.py:937]:
```suggestion
        self.end_date: Optional[datetime.datetime] = self._resolver(
```

**Comment by bain3** [comment]
Hmm... I'm not sure whether to keep or remove `Information`s that do not have a start and end date when the user requests `Information`s from a selected period. I'd probably say to remove them, but it is kinda weird.

**Comment by Bapt5** [comment]
Do as you prefer for the `Information` without date. For the test I think we should try to run it again.  It should work

**Review by bain3 — APPROVED** [review]

**Pull request merged**